src/building-blocks/NestedFieldSet.tsx

```diff
@@ -5,17 +5,12 @@ import autoBindMethods from 'class-autobind-decorator';
 
 import { splitName } from '@mighty-justice/utils';
 
-import {
-  fillInFieldSet,
-  FormFieldSet,
-  IFieldConfigPartial,
-  IFieldSetPartial,
-  mapFieldSetFields,
-} from '../';
-
-import { FormManager } from '../utilities';
+import { fillInFieldSet, FormManager, mapFieldSetFields } from '../utilities';
```

Comment:
obviously looks better like this but why was it originally like line 8-14? What was the purpose?

Reply:
8-14 just imports everything from `index.ts`, like so:

![image](https://user-images.githubusercontent.com/796717/58585563-0de5a500-8227-11e9-832c-b98ed1dc982c.png)

This is fine? I don't know. I don't feel strongly one way or the other. But Storybook has problems with imports like this, because it's not as smart as Babel/Webpack, so I switched the file to these separate imports.